bookkeeper-server/src/test/resources/create.certificates.sh

```diff
@@ -0,0 +1,33 @@
+#/bin/bash
```

Comment:
What´s the purpose of this script? Testing? I'm wondering whether this is something we need to check in.